### PR TITLE
Add Device Name column to Route report

### PR DESCRIPTION
**Pull request by Abyss777**
Reused function as AttributeFormatter
fix https://github.com/tananaev/traccar/issues/2554

### CODE REVIEW

File: web/app/AttributeFormatter.js [hunk 1]
```diff
@@ -47,6 +47,10 @@ Ext.define('Traccar.AttributeFormatter', {
         return (hours + ' ' + Strings.sharedHourAbbreviation + ' ' + minutes + ' ' + Strings.sharedMinuteAbbreviation);
     },
 
+    deviceNameFormatter: function (value) {
+        return Ext.getStore('Devices').findRecord('id', value).get('name');
```
Comment (tananaev) [inline web/app/AttributeFormatter.js]: There is a special method `getById` or something like that. Potentially it's more efficient.

File: web/app/AttributeFormatter.js [hunk 2]
```diff
@@ -78,6 +82,8 @@ Ext.define('Traccar.AttributeFormatter', {
             return this.hoursFormatter;
         } else if (key === 'duration') {
             return this.durationFormatter;
+        } else if (key === 'deviceName') {
```
Comment (tananaev) [inline web/app/AttributeFormatter.js]: Key should be `deviceId`. That's the naming convention to have the same name as the model key.

**Comment by Abyss777** [comment]
Sorry. Fixed.

**Comment by tananaev** [comment]
Merged, thanks. I guess this one goes into the next release.

**Pull request merged**